resources/definitions/modix_base.def.json

```
@@ -0,0 +1,198 @@
+		"machine_gcode_flavor": {"default_value": "RepRap (RepRap)"},
+
+        "machine_heated_bed": { "default_value": false },
+
+        "material_diameter": { "default_value": 1.75 },
+		
+        "machine_head_with_fans_polygon": { "default_value": [[-100,30],[-100,65],[30,-105],[-31,-100]] },
+        "gantry_height": {"value": "90.0"},
+
+        "acceleration_enabled": { "value": false },
+        "jerk_enabled": { "value": false },
+
+        "speed_print": { "value": 60.0 } ,
+        "speed_infill": { "value": "speed_print" },
+        "speed_wall": { "value": "speed_print / 2" },
+        "speed_wall_0": { "value": "speed_wall" },
+        "speed_wall_x": { "value": "speed_wall" },
+        "speed_topbottom": { "value": "speed_print / 2" },
+        "speed_roofing": { "value": "speed_topbottom" },
+        "speed_travel": { "value": "150.0 if speed_print < 60 else 250.0 if speed_print > 100 else speed_print * 2.5" },
+        "speed_layer_0": { "value": 20.0 },
+        "speed_print_layer_0": { "value": "speed_layer_0" },
+        "speed_travel_layer_0": { "value": "100 if speed_layer_0 < 20 else 150 if speed_layer_0 > 30 else speed_layer_0 * 5" },
+        "speed_prime_tower": { "value": "speed_topbottom" },
+        "speed_support": { "value": "speed_wall_0" },
+        "speed_support_interface": { "value": "speed_topbottom" },
+        "speed_z_hop": { "value": 5 },
+
+        "skirt_brim_speed": { "value": "speed_layer_0" },
+
+        "line_width": { "value": "machine_nozzle_size * 1.1" },
+
+        "optimize_wall_printing_order": { "value": "True" },
+
+        "material_initial_print_temperature": { "value": "material_print_temperature" },
+        "material_final_print_temperature": { "value": "material_print_temperature" },
+        "material_flow": { "value": 100 },
+        "travel_compensate_overlapping_walls_0_enabled": { "value": "False" },
+
+        "z_seam_type": { "value": "'back'" },
+        "z_seam_corner": { "value": "'z_seam_corner_none'" },
+        
+        "infill_sparse_density": { "value": "20" },
+        "infill_pattern": { "value": "'lines' if infill_sparse_density > 50 else 'cubic'" },
+        "infill_before_walls": { "value": false },
+        "infill_overlap": { "value": 30.0 },
+        "skin_overlap": { "value": 10.0 },
+        "infill_wipe_dist": { "value": 0.0 },
+        "wall_0_wipe_dist": { "value": 0.0 },
+
+        "fill_perimeter_gaps": { "value": "'everywhere'" },
+        "fill_outline_gaps": { "value": false },
+        "filter_out_tiny_gaps": { "value": false },
+
+        "retraction_speed": {
+            "maximum_value_warning": "machine_max_feedrate_e if retraction_enable else float('inf')",
```

```suggestion
            "maximum_value_warning": "machine_max_feedrate_e if retraction_enable else float('inf')",
```
No need to set this; The default is already the same (DRY: Don't Repeat Yourself)

---

There are other cases where this is also the case. I've not marked all of them.